cmd/cli/cmd/deploy.go

```diff
@@ -218,16 +222,24 @@ func appendE(slice []string, arg string, val string) []string {
 }
 
 func checkDockerExists() {
-	_, err := exec.LookPath("docker")
+	_, err := exec.LookPath(getContainerEngineForBuild())
 	if err != nil {
-		fmt.Printf("please add 'docker' to the path\n")
+		fmt.Printf("please add '%s' to the path\n", getContainerEngineForBuild())
 	}
 }
 
 func checkInitialized() bool {
 	return InitAction()
 }
 
+func getContainerEngineForBuild() string {
+	if viper.GetString("container_engine") != "" {
```

Comment:
Flip flop these returns, for consistency we want to respect cli arg present over declarative config (yaml) present.